tb_plugin/fe/src/utils/top.ts

```diff
@@ -30,8 +30,13 @@ export function useTopN(options?: IOptions) {
     ? React.useCallback(debounce(setActualTop, options.wait ?? 500), [])
     : setActualTop
   React.useEffect(() => {
-    setActualDebounce(useTop === UseTop.Use && top > 0 ? top : undefined)
-  }, [top, useTop])
+    setActualDebounce(useTop === UseTop.Use && topIsValid(topText) ? Number(topText): actualTop)
```

Comment:
in case of not valid, what the actualtop will be?

Reply:
The previous valid value.  If the user enters something invalid, the graph will not be updated until a valid value is provided.